content/docs/start/experiments-trail/experiments.md

```diff
@@ -0,0 +1,319 @@
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '8bb6049' for future execution.
+```
+
+Next, run all (`--run-all`) queued experiments in parallel. You can specify the
+number of parallel processes using `--jobs`:
+
+```dvc
+$ dvc exp run --run-all --jobs 2
+```
+
+## ↔️ Comparing experiments
+
+The experiments are run several times with different parameters. We use
+`dvc exp show` to compare all of these experiments. This command presents the
+parameters and metrics produced in experiments in a nicely formatted table.
+
+```dvc
+$ dvc exp show
+```
+
+![](/img/start-dvc-exp-show-210704.png)
+
+By default it shows all the parameters and the metrics with the timestamp. If
+you have large number of parameters, metrics or experiments, this may lead to a
+cluttered view. You can limit the table to specific metrics, or parameters, or
+hide the timestamp column with `--include-metrics`, `--include-params`, or
+`--no-timestamp` options of the command, respectively.
+
+```dvc
+$ dvc exp show --no-timestamp \
+  --include-params model.conv_units --include-metrics acc
+```
+
+![](/img/start-dvc-exp-show-no-timestamp-210704.png)
+
+## 🔏 Persisting experiments
+
+After selecting an experiment from the table, you can commit the hyperparameters
+and other dependencies that produced this successful experiment to your Git
+history.
+
+`dvc exp apply` brings back all specific artifacts and parameters from the
+experiment to the <abbr>workspace</abbr>.
+
+```dvc
+$ dvc exp apply exp-cb13f
+Changes for experiment 'exp-cb13f' have been applied to your current workspace.
+```
+
+We can see the changes in the repository and commit them to Git.
+
+```dvc
+$ git diff
+$ git add .
+$ git commit -m "Successful experiment"
+```
+
+It may also be desirable to commit a particular experiment to a Git branch
+directly, without bringing to the workspace.
+
+```dvc
+$ dvc exp branch exp-05e87 "cnn-256"
+Git branch 'cnn-256' has been created from experiment 'exp-05e87'.
+To switch to the new branch run:
+
+        git checkout cnn-256
+```
+
+You can then checkout and continue from working this branch as usual.
+
+## 📛 Note on experiment names
+
+When you create an experiment, DVC generates a hash value from the contents of
+the experiment. This is shown when you use `--queue` option, e.g.,
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+```
+
+After _running_ the experiment, DVC uses another auto-generated name to refer to
+the experiment. Typically these start with `exp-`, and can be set via
+`--name / -n` option of `dvc exp run`. So when you add an experiment by setting
+the name, you can see the hash value as _queued experiment_:
+
+```dvc
+$ dvc exp run --queue --name cnn-512 -S model.conv_units=512
+Queued experiment '86bd8f9' for future execution.
+```
+
+In `dvc exp show` you can see both of these names:
+
+```dvc
+$ dvc exp show --no-pager --no-timestamp \
```

Comment:
we need to fix the highlighter? cc @jorgeorpinel

Reply:
Why? What's wrong? Sorry, review app isn't up rn.